packages/semantic-release-config/index.js

```diff
@@ -1,43 +1,55 @@
 module.exports = {
   plugins: [
     [
-      "@semantic-release/commit-analyzer",
+      '@semantic-release/commit-analyzer',
       {
-        preset: "conventionalcommits",
+        preset: 'conventionalcommits',
         releaseRules: [
-          { type: "docs", release: "patch" },
-          { type: "refactor", release: "patch" },
-          { type: "style", release: "patch" },
-          { type: "test", release: "patch" },
-          { type: "ci", release: "patch" },
-          { type: "build", release: "patch" },
+          { type: 'docs', release: 'patch' },
+          { type: 'refactor', release: 'patch' },
+          { type: 'style', release: 'patch' },
+          { type: 'test', release: 'patch' },
+          { type: 'ci', release: 'patch' },
+          { type: 'build', release: 'patch' },
         ],
         parserOpts: {
-          noteKeywords: ["BREAKING CHANGE", "BREAKING CHANGES", "BREAKING"],
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING'],
         },
       },
     ],
     [
-      "@semantic-release/release-notes-generator",
+      '@semantic-release/release-notes-generator',
       {
-        preset: "conventionalcommits",
+        preset: 'conventionalcommits',
         parserOpts: {
-          noteKeywords: ["BREAKING CHANGE", "BREAKING CHANGES", "BREAKING"],
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING'],
+        },
+        parserConfig: {
+          types: [
+            { type: 'feat', section: 'Features' },
+            { type: 'fix', section: 'Bug Fixes' },
+            { type: 'chore', hidden: true },
+            { type: 'docs', section: 'Documentation' },
+            { type: 'style', hidden: true },
+            { type: 'refactor', hidden: true },
```

Comment:
If we're releasing on `style` and `refactor` commits (as configured in the `releaseRules` section above), shouldn't the commits be in the release notes are well?

Reply:
yes, probably they should